include/SampleCache.h

```diff
@@ -0,0 +1,37 @@
+/*
+ * SampleCache.h - A class for the sample cache
+ *
+ * Copyright (C) 2022 JGHFunRun <JGHFunRun@gmail.com>
+ *
+ * This file is part of LMMS - https://lmms.io
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public
+ * License as published by the Free Software Foundation; either
+ * version 2 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this program (see COPYING); if not, write to the
+ * Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+ * Boston, MA 02110-1301 USA.
+ *
+ */
+
+#ifndef SAMPLE_CACHE_H
+#define SAMPLE_CACHE_H
```

Comment:
perhaps you can use CamelCase instead of all caps for better readability

Reply:
That would be inconsistent with the rest of the codebase.